frontend/src/components/activePage/utils/data.tsx

```diff
@@ -0,0 +1,53 @@
+export interface IVM {
```

Comment:
Since you use this file inside a single component, decelare the needed interface there and if needed, import them in the stories